charts/common/values.yaml

```diff
@@ -0,0 +1,596 @@
+      #overrideUserKey: custom-username
+      # overridePasswordKey is used for overriding the default passwordKey (.Values.servicemonitor.basicAuth.passwordKey)
+      #overridePasswordKey: custom-password
+      # componentSelector is used for selecting specific pods
+      #componentSelector:
+        # key is the name of the label
+        #key: app.kubernetes.io/component
+        # value is the value of the label
+        #value: custom-komponente
+      # headlessTargetPort is the port for the headless service. Comment out for using default ("8080")
+      #headlessTargetPort: 8080
+      # path is the path to scrape metrics from. Comment out for using default ("/metrics")
+      #path: /metrics
+      # interval sets the scrape interval. Comment out for using default ("30s")
+      #interval: 30s
+      # port is used for matching the headless service. Comment out for using default ("http")
+      #port: http
+# end common.servicemonitor
+
+# start common.pvcs
+# pvcs is used to create pvcs outside of deployments/statefulsets and is not bound to components
+pvcs:
+  # specify the name of the pvc
+  #- name: name-of-pvc
+  # size is optional with a default value of "1Gi"
+    #size: 10Gi
+  # accessModes is an optional list with a default list with the following slice: "ReadWriteOnce"
+    #accessModes:
+    #  - ReadWriteMany
+  # storageClassName is optional with a default value of ""
+    #storageClassName: "nfs-data"
+# end common.pvcs
+
+# components are a dictionary of single components to be configured
+components:
+  # specify the name of the component
+  component-1:
+    # start common.service
+    services:
+      # specify the name of the service
+      service-1:
+        # deploy has to be set to true for rendering to be applied
+        deploy: false
+        # type is optional with a default value of "ClusterIP"
+        #type:
+        # name is optional with a default value of "http"
+        #name:
+        # port is optional with a default value of "80"
+        #port:
+        # targetPort is optional with a default value of "http"
+        #targetPort:
+        # protocol is optional with a default value of "TCP"
+        #protocol:
+        # overrideSelectors is optional for overridding the default labels: 'app.kubernetes.io/name' & 'app.kubernetes.io/component'
+        #overrideSelectors:
+        #  label2.1: "label2"
+        #  label2.2: "label2"
+        # sessionAffinity supports "ClientIP" and "None". Used to maintain session affinity. Default value is "None".
+        #sessionAffinity: "ClientIP"
+    # end common.service
+    # start common.networkpolicy
+    # networkpolicy allows access from outside of the namespace or from other pods
+    networkpolicy:
+      # deploy has to be set to true for rendering to be applied
+      deploy: false
+      # trusted is optional with a default value of "false"
```

Comment:
if we keep the trusted function, add this comment to clarify the effects
`, it allows traffic from namespaces with label "trusted" (e.g. for system namespaces like ingress-controller)`